### ISSUE TITLE
Unprocessing Images for Learned Raw Denoising

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#18]
## Main Idea
**Denoising through realistic image processing: Unprocess**
Paper: <https://arxiv.org/abs/1811.11127>

## Abstract
Machine learning techniques work best when the data used for training resembles the data used for evaluation. This holds true for learned single-image denoising algorithms, which are applied to real raw camera sensor readings but, due to practical constraints, are often trained on synthetic image data. Though it is understood that generalizing from synthetic to real images requires careful consideration of the noise properties of camera sensors, the other aspects of an image processing pipeline (such as gain, color correction, and tone mapping) are often overlooked, despite their significant effect on how raw measurements are transformed into finished images. To address this, we present a technique to “unprocess” images by inverting each step of an image processing pipeline, thereby allowing us to synthesize realistic raw sensor measurements from commonly available Internet photos. We additionally model the relevant components of an image processing pipeline when
evaluating our loss function, which allows training to be aware of all relevant photometric processing that will occur after denoising. By unprocessing and processing training data and model outputs in this way, we are able to train a simple convolutional neural network that has 14%-38% lower error rates and is 9×-18× faster than the previous state of the art on the Darmstadt Noise Dataset, and generalizes to sensors outside of that dataset as well.

## Contribution
+ Fake image의 생성에 있어서 무분별한 가정을 통한 생성이 아닌, 광학적인 접근을 통해 denoising의 문제를 더 low-level task로 이끔
+ Denoising에 있어서 deep learning시에 일반적으로 인공 학습 데이터에 의존해 대부분의 학습이 이루어짐 => 성능의 저하
+ 이를 극복하기 위해 카메라가 수행하는 image processing의 과정을 단계적으로 역변환하는 'unprocess'의 방안을 제시
+ Training Process
 1. 카메라는 raw image를 자체적으로 내장된 여러가지 광학적인 보정기능을 거친 뒤, sRGB형태로 반환
 2. sRGB image를 여러가지 'unprocessing'의 과정을 통해 raw image로 변환
 3. shot noise와 read noise를 추가해 noisy raw image를 생성
 4. 이를 noise lebel map과 함께 신경망에 통과시켜 denoised raw image를 형성
 5. 다시 'processing'의 과정을 통해 카메라가 제공하는 기능들을 다시 입힘
 6. 출력된 sRGB형태의 image와 raw image에 곧바로 processing한 noise-free sRGB image(ground truth)와 비교해 L1 loss를 계산
+ U-net기반의 sRGB로부터 계산한 모델과 raw로부터 계산한 모델 두 가지를 사용
+ 가장 높은 성능과 함께 과거 모델들보다 9-18배 빠른 속도

## Reviews
<https://deepkerry.tistory.com/31>
